brokerbot.go

```diff
@@ -141,19 +141,28 @@ func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
 	expandedString := messagelib.ExpandAliases(userInput)
 	tickers := messagelib.DedupeTickerStrings(strings.Split(expandedString, " "))
 
+	cryptoContext, cancelFunc := context.WithCancel(ctx)
+	priceFeeds, err := cryptolib.GetPriceFeeds()
+	if err != nil {
+		log.Printf("Failed to fetch Price Feeds: %s", err);
```

Comment:
Should this return an error? What happens if the feeds are nil in the following logic?

Reply:
We log that the Price Feeds failed, and then we return the Ticker with "no data" similar to how the behavior works. I think this is the best we can do, because the request might include a mix of Stocks and Crypto, so we should still proceed for Stocks.